src/components/modal/Modal.scss

```diff
@@ -153,6 +153,15 @@
         .modal-close-button {
             right: 10px;
         }
+
+        .modal-actions {
+            position: fixed;
```

Comment:
It doesn't seem ideal to have nested fixed position elements. Can we utilize flexbox or relative/absolute positioning, instead?

Reply:
Hi Jared! I've been looking for an alternate way to do this trying not u sed the position: fixed, but been unable to come with a simple solution like this one. I know the fixed button container is nested and that might not seem ideal, but this would only be for the modal in small screens. If you have some other options I'll appreciate it.

Reply:
@msorianoBox  settting `.modal-content` to `flex-grow: 1; position: relative;` should do the trick for the container part^

Reply:
@msorianoBox and I went over this on a call. The main issue is that `.modal-actions` can be nested to an arbitrary depth within `.modal-content`, so we can't rely on flexbox or relative positioning at the component level. Apart from refactoring Modal's layout styles, I'm not sure there's a better option than nested fixed positions right now.

Reply:
Ah yeah you're right.

Question about using `position: fixed`:
For a modal that has content (`.modal-content`) that expands the entire height of the screen, wouldn't a fixed position for `.modal-actions` block/cover the bottom of that content? I think we might need to add a padding to `.modal-content` that takes into account the height of `.modal-actions`.

Reply:
@tjuanitas Done :D